[2024-04-04T11:43:37.283+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag___fresh.handle_dbt_result manual__2024-04-04T11:43:05.161204+00:00 [queued]>
[2024-04-04T11:43:37.293+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag___fresh.handle_dbt_result manual__2024-04-04T11:43:05.161204+00:00 [queued]>
[2024-04-04T11:43:37.294+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2024-04-04T11:43:37.312+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): handle_dbt_result> on 2024-04-04 11:43:05.161204+00:00
[2024-04-04T11:43:37.318+0000] {standard_task_runner.py:60} INFO - Started process 6959 to run task
[2024-04-04T11:43:37.322+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'dag___fresh', 'handle_dbt_result', 'manual__2024-04-04T11:43:05.161204+00:00', '--job-id', '407', '--raw', '--subdir', 'DAGS_FOLDER/dbt_dag.py', '--cfg-path', '/tmp/tmppo5g3slr']
[2024-04-04T11:43:37.325+0000] {standard_task_runner.py:88} INFO - Job 407: Subtask handle_dbt_result
[2024-04-04T11:43:37.375+0000] {task_command.py:423} INFO - Running <TaskInstance: dag___fresh.handle_dbt_result manual__2024-04-04T11:43:05.161204+00:00 [running]> on host 62dc842f1ceb
[2024-04-04T11:43:37.463+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag___fresh' AIRFLOW_CTX_TASK_ID='handle_dbt_result' AIRFLOW_CTX_EXECUTION_DATE='2024-04-04T11:43:05.161204+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-04T11:43:05.161204+00:00'
[2024-04-04T11:43:37.466+0000] {logging_mixin.py:188} INFO - Result of dbt command:
[2024-04-04T11:43:37.466+0000] {logging_mixin.py:188} INFO - -------
[2024-04-04T11:43:37.477+0000] {logging_mixin.py:188} INFO - {'elementary_freshness_anomalies_raw_orders_creation_date': 'PASS', '1': 'FAIL', 'not_null_my_second_dbt_model_id': 'PASS', 'not_null_raw_orders_creation_date': 'PASS', 'unique_my_first_dbt_model_id': 'PASS', 'unique_my_second_dbt_model_id': 'PASS'}
[2024-04-04T11:43:37.478+0000] {logging_mixin.py:188} INFO - -----------
[2024-04-04T11:43:37.479+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-04-04T11:43:37.490+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=dag___fresh, task_id=handle_dbt_result, execution_date=20240404T114305, start_date=20240404T114337, end_date=20240404T114337
[2024-04-04T11:43:37.535+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-04-04T11:43:37.554+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
